src/server/game/Scripting/ScriptObject.cpp

```diff
@@ -0,0 +1,377 @@
+/*
+ * This file is part of the WarheadCore Project. See AUTHORS file for Copyright information
```

Comment:
And here.

Reply:
Found a few more, but I don't want to clog up your PR. I'll wait until you take this out of DRAFT and then track down any remaining ones.